modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -257,6 +257,12 @@ const EditorSelection = (dom: DOMUtils, win: Window, serializer: DomSerializer,
     return !sel || rng.collapsed;
   };
 
+  /**
+   * Checks if the current selection start and end containers are editable within their parent contexts.
```

Comment:
```suggestion
   * Checks if the current selection’s start and end containers are editable within their parent’s contexts.
```
See [my notes in Notion](https://notion.so/tinycloud/Notes-on-possessive-apostrophes-vs-contraction-apostrophes-5dfb8def1b054de28e4ad5d5cbe63631) for all the gory details behind this tiny edit.